less/tabzilla.less

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "lib/lib.less";
```

Comment:
True, although I think it will likely add maybe extra maintenance to this repo, since many of these things will likely change in bedrock and be forgotten about here. Personally I'd vote for only including what we need :+1: